Query/Event/PreFilterConditionCreationEvent.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Netdudes\DataSourceryBundle\Query\Event;
+
+use Netdudes\DataSourceryBundle\DataType\DataTypeInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class PreFilterConditionCreationEvent extends Event
+{
+    /**
+     * @var DataTypeInterface
+     */
+    private $dataType;
+
+    /**
+     * @var mixed
+     */
+    private $value;
```

Comment:
`$valueInDatabase`?
Should user of this event know that this is value in database or for him it's only a value? What do you think?